zulipterminal/model.py

```diff
@@ -1381,18 +1381,19 @@ def _handle_typing_event(self, event: Event) -> None:
         if (
             len(narrow) == 1
             and narrow[0][0] == "pm_with"
-            and sender_email in narrow[0][1].split(",")
+            and sender_email in list(narrow[0][1].split(", "))
             and sender_id != self.user_id
         ):
             if event["op"] == "start":
-                sender_name = self.user_dict[sender_email]["full_name"]
-                active_conversation_info["sender_name"] = sender_name
+                sender_email = self.user_dict[sender_email]["email"]
```

Comment:
Yeah agreed. When I wrote the next line `active_conversation_info.add(sender_email)`, I was thinking that sender_email is the variable that we are talking about now to remove and not the previously declared `sender_email = event["sender"]["email"]`.
I think this can also be done in the `event["op"] == "stop"`